README.md

```diff
@@ -405,10 +517,30 @@ candidates_pd = \
                      number_test_rows=20)
 ```
 
+Similarly, if `output_par_dir="my_combined_dir"` was set in [combine vector representations](#combine-vector-representations), we need to set `scenario="./my_combined_dir/test"`: 
+
+```python
+from DeezyMatch import candidate_ranker
+
+# Find candidates
+candidates_pd = \
+    candidate_ranker(scenario="./my_combined_dir/test", 
+                     ranking_metric="faiss", 
+                     selection_threshold=5., 
+                     num_candidates=1, 
+                     search_size=4, 
+                     output_filename="test_candidates_deezymatch", 
+                     pretrained_model_path="./models/finetuned_test001/finetuned_test001.model", 
+                     pretrained_vocab_path="./models/finetuned_test001/finetuned_test001.vocab", 
+                     number_test_rows=20)
+```
+
```

Comment:
I think lines 520-537 are unnecessary: we need to specify the scenario folder anyway. We already say in combining vectors that we can specify the output folder. I think we only need to remind here that `scenario` should contain the path to the output folder specified in `combine_vecs`.

Reply:
OK, here https://github.com/Living-with-machines/DeezyMatch/commit/5b08ac495a28b8b64f7a0cd2de9228374f0780ce. I need to properly review/edit the README file once we are happy with the new features/functionalities.